apps/server/src/services/report-service/ReportService.ts

```
@@ -0,0 +1,62 @@
+    }
+    return;
+  }
+
+  // this clears out potentaly old data
+  report.set(state.eventNow.id, { ...blankReportData, startAt: state.timer.startedAt });
+}
+
+export function eventStop(state: RuntimeState) {
+  if (state.eventNow === null) {
+    // eslint-disable-next-line no-unused-labels -- dev code path
+    DEV: {
```

Biome

[error] Unexpected label.

Only loops should be labeled.
The use of labels for other statements is suspicious and unfamiliar.

(lint/suspicious/noConfusingLabels)